pkg/brctl/brctl.go

```diff
@@ -179,7 +180,12 @@ func Showmacs(bridge string, out io.Writer) error {
 
 // Show will show some information on the bridge and its attached ports.
 func Show(out io.Writer, names ...string) error {
-	fmt.Fprint(out, "bridge name\tbridge id\tSTP enabled\t\tinterfaces")
+	fmt.Fprintf(out, "%-15s %23s %15s %20s\n",
```

Comment:
My personal preference is not to wrap lines like this, and I know Rob would be on my side :-)
This seems fine otherwise. A very nice catch.
i.e. make this one line. Certainly is easier when grepping source.

Reply:
you might, to ensure formatting never changes, make a function:
var printbridge(w io.Writer, name, id, state, face string) {
fmt.Fprintf(out, etc.)
}
OR make a
const ShowBridgeFmt = "%-15s %23s %15v %20v\n"
because someone is sure to change one and miss the other string, someday.

Reply:
Agree, however the line is not too long, so I removed the line breaks here. 
Abstracting the formatting is a good hint, added that.